rules/ml/ml_linux_system_user_discovery.toml

```
@@ -0,0 +1,40 @@
+[metadata]
+creation_date = "2020/09/03"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/09/03"
+
+[rule]
+anomaly_threshold = 75
+author = ["randomuserid"]
```

```suggestion
author = ["Elastic"]
```
All ML rules should be updated to `Elastic` as author